docs/extras/faq.md

```diff
@@ -1,14 +1,18 @@
 # Frequently Asked Questions {docsify-ignore-all}
 ---
+
+### Does this work on the latest version?
+Yes, at the time of reading this it's at 5.5.5
```

Comment:
Not sure if this is really needed in the FAQ.
The Introduction page already says `This guide will work on all Wii U consoles in all regions on firmware 5.5.5 or below.`.

Reply:
Alright, i should probably remove it.